qml/harbour-berlin-vegan.qml

```diff
@@ -161,6 +161,7 @@ ApplicationWindow
 
             onClicked: {
                 gJsonCollection.filterFavorites = true;
+                gJsonCollection.filterVenueType = VenueModel.FoodFlag | VenueModel.ShoppingFlag;
```

Comment:
Please explain, why this line is necessary.

Reply:
I extracted the test for a favorites from the venueTypeMatches() helper function in its own helper function

    favoriteStatusMatches().

But now we need to explicitly set

    gJsonCollection.filterVenueType = VenueModel.FoodFlag | VenueModel.ShoppingFlag;

in the favorites menu item, otherwise not both venue types would be shown in the favorites list.

B4, the test was simply aborted when favorites was true in venueTypeMatches()